newm-server/src/main/kotlin/io/newm/server/features/release/model/SpotifyRequest.kt

```diff
@@ -9,4 +9,22 @@ data class SpotifyRequest(
     val uris: Array<String>,
     @SerialName("position")
     val position: Int,
-)
+) {
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+
+        other as SpotifyRequest
+
+        if (!uris.contentEquals(other.uris)) return false
+        if (position != other.position) return false
+
+        return true
+    }
+
+    override fun hashCode(): Int {
+        var result = uris.contentHashCode()
+        result = 31 * result + position
+        return result
+    }
```

Comment:
You shouldn't need to add these. They are generated automatically for Kotlin Data classes. What's the issue you're seeing?

Reply:
@wlara  
I changed based on your recommendation, for more information about the issue:  https://projectnewm.atlassian.net/browse/STUD-291 

Let me know what you think about it

Reply:
To fix the serialization crash, you need to change from `setBody {...}` to `setBody(..)` on the Ktor builder of the Network call. More details via Slack.